### DISCUSSION TITLE
Alerting: Ability to add timezone for Alertmanager contact point

### DISCUSSION THREAD

**Discussion by Jessimon** [grafana/grafana#49395] [Alerting]
Hello Grafana,

I have the following setup:
Grafana 8.5.2 running on the local system.
Alertmanager 0.23 running in docker.

Both the local system and the docker OS timezone are set to Europe/Brussels.
Grafana logs show the correct local time.
Alertmanager logs shows the UTC time.

The issue is that the Alerts are generated by Grafana unified alerting and are then send to Alertmanager to handle the notification.
Currently when an alert is send from Grafana unified alerting we get a resolved message after group_interval from the Alertmanager even if that alert is not resolved.

So fare have not found a way to change the timezone in Alertmanager.
In datasources you can add the timezone, but not for contactpoints. (Probably not needed for most of them)

Thanks for the help.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
